### ISSUE TITLE
Third-party Implementation in PyTorch

### ISSUE THREAD

**Issue by d-li14** [facebookresearch/OctConv#2]
Just for reference, there is a PyTorch implementation of OctConv
https://github.com/d-li14/octconv.pytorch, including detailed training logs and pre-trained models of ResNet-50 on the ImageNet benchmark.

**Comment by cypw** [comment]
@d-li14 
Wow, you actually got better accuracy than the one reported in our paper!

In our paper:
- ResNet-50 (baseline): 77.0%
- Oct-ResNet-50 (alpha=0.5): 77.3%

https://github.com/d-li14/octconv.pytorch:
- ResNet-50 (baseline): 77.15%
- Oct-ResNet-50 (alpha=0.5): 77.64%

Thank you for your efforts. I've included a link to your GitHub repo.

**Comment by d-li14** [comment]
@cypw Yeah, thanks for giving me credit, as well as your awesome original implementation!
